### PR TITLE
Updates to Power BI against master branch

### PR DESCRIPTION
**Pull request by JeremyLikness**
Added description, fixed naming, and linked to samples per team feedback. Tagging @camsoper (I closed the request that was incorrectly against live) and @guyinacube as FYI. Will have team review for sign-off.

### CODE REVIEW

**Comment by openpublishbuild** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/powerbi-embedded.md](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-3/api/overview/azure/powerbi-embedded.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/overview/azure/powerbi-embedded?branch=pr-en-us-286) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C7%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201708071306473075-286%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api/overview/azure/powerbi-embedded.md [hunk 1]
```diff
@@ -1,39 +1,78 @@
...
+### Example
+
+The following example retrieves and displays a list of datasets and reports.
+
+```csharp
+/* Include these'using' directive:
+using Microsoft.PowerBI.Api.V2;
+using Microsoft.PowerBI.Api.V2.Models;
+*/
+using (var client = new PowerBIClient(new Uri(apiUrl), tokenCredentials))
```
Comment (spboyer) [inline api/overview/azure/powerbi-embedded.md]: I we not suppose to be using `var` here?

**Comment by spboyer** [comment]
#sign-off

**Comment by spboyer** [comment]
#sign-off

**Comment by openpublishbuild** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/powerbi-embedded.md](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-3/api/overview/azure/powerbi-embedded.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/overview/azure/powerbi-embedded?branch=pr-en-us-286) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C8%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201708081226404086-286%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**